content/blog/2022-08-29-announcing-poetry-1-2-0.md

```diff
@@ -0,0 +1,752 @@
+---
+layout: single
+title: "Announcing Poetry 1.2.0"
+date: 2022-08-23 00:00:00
+categories: [releases]
+tags: ["1.x", "1.2"]
+---
+
+The Poetry team is pleased to announce the immediate availability of Poetry 1.2.0.
+
+<!--more-->
+
+Poetry 1.2 boasts a massive list of changes, new features and bugfixes developed over the course of 2 years, with
+contributions from dozens of contributors.
```

Comment:
```suggestion
changes from dozens of contributors.
```

Reply:
now you repeat "changes" with "(...)a massive list of changes, new(...)"... I would leave it as it was before, with "contributions".

Reply:
> now you repeat "changes" with "(...)a massive list of changes, new(...)"... I would leave it as it was before, with "contributions".

Or perhaps a slight tweak along the lines of: "… contributions from dozens of people." 🚲 🏡